aviary/mission/flops_based/phases/two_dof_phase.py

```diff
@@ -77,29 +77,20 @@ def build_phase(self, aviary_options: AviaryValues = None):
 
         user_options: AviaryValues = self.user_options
 
-        throttle_setting = user_options.get_val('throttle_setting')
         control_order = user_options.get_val('control_order')
```

Comment:
Probably. Since it's used in multiple places and will be available even if `use_polynomial_control` is False, I'll keep it here. n.b.: this keyword changes in #166